src/test/java/org/lmdbjava/EnvTest.java

```diff
@@ -338,7 +339,7 @@ public void setMapSize() throws IOException {
     final Random rnd = new Random();
     try (Env<ByteBuffer> env = create()
         .setMaxReaders(1)
-        .setMapSize(50_000)
+        .setMapSize(KIBIBYTES.toBytes(256))
```

Comment:
Thanks @benalexau - this test succeeded on Apple Silicon for the wrong reason - I've made these changes with a new cut-off and tested locally on Linux/x86-64 and Mac/aarm64. It will be good to see if passes on the GH workflow now.